### ISSUE TITLE
get_orbit_number raising AttributeError

### ISSUE THREAD

**Issue by lmar76** [ESA-VirES/VirES-Python-Client#40]
The `SwarmRequest.get_orbit_number` method raises `AttributeError`:

```
In [5]: req.get_orbit_number('A', dt.datetime(2019, 9, 12, 0, 52, 54, 583987))                                          
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-585e7c547a7c> in <module>
----> 1 req.get_orbit_number('A', dt.datetime(2019, 9, 12, 0, 52, 54, 583987))

/usr/local/opt/anaconda3/envs/swarm/lib/python3.7/site-packages/viresclient/_client_swarm.py in get_orbit_number(self, spacecraft, input_time)
    751         retdata = ReturnedDataFile(filetype="csv")
    752         response_handler = self._response_handler(
--> 753             retdata.file, show_progress=False
    754         )
    755         self._wps_service.retrieve(request, handler=response_handler)

AttributeError: 'ReturnedDataFile' object has no attribute 'file'
```

**Comment by pacesm** [comment]
fixed in 49bd5f289c515f67e4ceb38d2e02a6daa38c6988

**Comment by lmar76** [comment]
Tested.

**Comment by lmar76** [comment]
Could it be possible to generate a new SW delivery with this fix (e.g. v0.4.2)? Thanks.

**Comment by pacesm** [comment]
sure
@smithara, could you please release the current master head as v0.4.2? Thanks.

**Comment by smithara** [comment]
Done - released v0.4.2
